themes-default/slim/src/index.js

```diff
@@ -105,25 +109,16 @@ const UTIL = {
     }
 };
 
-$.fn.extend({
-    addRemoveWarningClass(_) {
```

Comment:
Heads up: it's still being used:
```
themes-default/slim/static/js/config/notifications.js
  123,29:             $('#emby_host').addRemoveWarningClass(emby.host);
  124,31:             $('#emby_apikey').addRemoveWarningClass(emby.apikey);
  164,36:             $('#pushover_userkey').addRemoveWarningClass(pushover.userkey);
  165,35:             $('#pushover_apikey').addRemoveWarningClass(pushover.apikey);
  199,27:         $('#twitter_key').addRemoveWarningClass(twitter.key);
  365,31:             $('#telegram_id').addRemoveWarningClass(telegram.id);
  366,35:             $('#telegram_apikey').addRemoveWarningClass(telegram.apikey);
  387,33:             $('#slack_webhook').addRemoveWarningClass(slack.webhook);
  438,34:             $('#trakt_username').addRemoveWarningClass(trakt.username);
```

Reply:
Not for long. Can be removed in the vueify of config_notifications

Reply:
It's not being used on my local branch since I already moved it but didn't commit that file. Sorry.